romana-install/roles/stack/kubernetes/install-minion/templates/cni/romana-k8s-network

```diff
@@ -2,8 +2,8 @@
   "name": "romana-k8s-network",
   "type": "romana",
   "kube_master_ip": "{{ romana_master_ip }}",
+  "romana_cli_config": "/home/{{ ansible_ssh_user }}/.romana.yaml",
```

Comment:
It'd be nice if the plugin didn't depend on things from `/home/`.
Can revisit this when we standardize some of the users/paths/etc.